packages/core/src/components/cms/RenderSections.tsx

```diff
@@ -109,12 +109,16 @@ const RenderSectionsBase = ({
 
         return (
           <SectionBoundary key={`cms-section-${name}-${index}`} name={name}>
-            <LazyLoadingSection
-              sectionName={name}
-              isInteractive={isInteractive}
-            >
+            {data.skipLazyLoadingSection ? (
```

Comment:
🪄 
Maybe in the future, we can expand this to all sections to control the renderization along with the ViewportObserver :)

Reply:
Yes!! We can explore it in a spike